builder.json

```
@@ -154,4 +170,4 @@
             "!build_env": {}
         }
     }
-}
+}
```

trivial: blank line at the end.
```suggestion
}

```